.github/workflows/build.yml

```diff
@@ -224,6 +226,26 @@ jobs:
         name: docker-image
         path: artifacts
 
+    - name: Generate sha256 checksums
+      if: github.event_name == 'release'
+      run: |
+        for filePath in $(find  artifacts/* -name '*.tar.gz' -or -name '*.zip')
+        do
+          fileName="$(basename -- $filePath)"
+          newSha=$(sha256sum "$filePath" | awk '{ print $1 }')
+          echo "$newSha $fileName" >> artifacts/OctopusTools.${{ needs.build.outputs.version }}.sha256sums
+        done
+
+    - name: Upload binaries and checksum file to release
+      if: github.event_name == 'release'
+      run: |
+        args=("v${{ needs.build.outputs.version }}")
+        for filePath in $(find  artifacts/* -name '*.tar.gz' -or -name '*.zip' -or -name '*.sha256sums')
+        do 
+          args+=($filePath)
```

Comment:
Add each individual file we want to upload to the release to the arguments to pass to `gh` (same as the ones that we calculated checksums for in the previous step)